.github/workflows/_create-release-pr.yml

```diff
@@ -53,9 +53,10 @@ jobs:
             || inputs.component-name == 'Compute' && 'release-compute'
           }}
       run: |
-        today=$(date +'%Y-%m-%d')
+        today=$(date -u +'%Y-%m-%d')
+        now=$(date -u +'%Y-%m-%d_%H-%M_UTC')
```

Comment:
nit: `--utc` flag to `date` will make it more clear that we use `UTC` time.
I also think in that case `_%Z` instead of `_UTC` should also work.

Reply:
> nit: --utc flag to date will make it more clear that we use UTC time.

Agree, but unfortunately BSD's (on macOS) `date` doesn't support `--utc`, I'll keep `-u` to make it easier to copy-and-run on any system

> I also think in that case _%Z instead of _UTC should also work.

Added